### ISSUE TITLE
Changelog upon new version

### ISSUE THREAD

**Issue by Phyks** [nrenner/brouter-web#421]
Hi,

There were many recent great additions to BRouter-web (which I missed, congrats for these!). What about keeping track of a CHANGELOG between versions and having it displayed automatically at first visit of Brouter-web?

It would probably help old users discover and get started with new features!

Best,

**Comment by bagage** [comment]
It's actually something that landed a few weeks ago in master (no release since), because indeed a lot of new features are coming and it's hard to follow/discover all of them :). Cf #372
